drivers/sensor/lsm6dsl/lsm6dsl_spi.c

```diff
@@ -0,0 +1,106 @@
+/* lsm6dsl_spi.c - SPI routines for LSM6DSL driver
+ */
+
+/*
+ * Copyright (c) 2018 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <spi.h>
+#include "lsm6dsl.h"
+
+#define LSM6DSL_SPI_READ		(1 << 7)
+
+#define SPI_CS NULL
+
+static struct spi_config lsm6dsl_spi_conf = {
+	.frequency = 16000000,
+	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
+	SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE,
+	.slave = CONFIG_LSM6DSL_SPI_SELECT_SLAVE,
+	.cs = SPI_CS,
+};
+
+static int lsm6dsl_raw_read(struct spi_config *spi_cfg, uint8_t reg_addr, uint8_t *value, uint8_t len)
+{
+	uint8_t buffer_tx[2] = { reg_addr | LSM6DSL_SPI_READ, 0 };
+	const struct spi_buf tx_buf = {
+			.buf = buffer_tx,
+			.len = 2,
+	};
+	uint8_t buffer_rx[64];
+	struct spi_buf rx_buf = {
+			.buf = buffer_rx,
+			.len = len + 1,
+	};
+
+	if (len > 64)
+		return -EIO;
```

Comment:
we enforce using {} always, even for one liners statement on if/for/while...

Apply that comment where relevant

Reply:
OK, I read the coding style exceptions to Linux now.
Thx!